game_eggs/steamcmd_servers/scpsl/exiled/README.md

```diff
@@ -0,0 +1,24 @@
+# SCP: Secret Laboratory Exiled
+SCP: Secret Laboratory Pterodactyl egg with Exiled Plugin Framework (https://github.com/Exiled-Team/EXILED)
+
+### Minimum RAM
+Minimum memory required to run the server: 4096 MB
+
+
+### Server Port
+Default port required to run the server.
+
+| Port    | Default |
+|---------|---------|
+| Game    | 7777    |
+
+
+### Verification
+
+The server must be verified with the SCP:SL developers in order for it to be visible in the server browser.
+
+Instructions are as follows:
+* Make sure your server complies with the [Verified Server Rules](https://scpslgame.com/Verified_server_rules.pdf).
+* Send an email to `server.verification@scpslgame.com` from your configured contact email with the following information:
+  1. The **public** IPv4 address of your server, with port. e.g. `69.69.69.69:7777`
+  2. If the server's IP is static or dynamic.
```

Comment:
This looks like an unfinished sentence?

Reply:
looks correct. but i never add this. the readme is c&p from dedicated